app/lib/features/comments/actions/sbumit_comment.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_easyloading/flutter_easyloading.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:logging/logging.dart';
+
+final _log = Logger('a3::submit::comment');
+
+Future<void> submitComment(
```

Comment:
```suggestion
Future<bool> submitComment(
```

Indicator whether the action was successful would be nice.